### PR TITLE
head error

### PR DESCRIPTION
**Pull request by abuts**
These are the simple changes to `head` which warn user about the fact he uses old format `head`. May be it worth running this format but the necessary changes are simple so I think the user would manage

### CODE REVIEW

File: horace_core/sqw/@sqw/head.m
Comment (oerc0122) [inline horace_core/sqw/@sqw/head.m:1]: All of these changes apart from those in this file are completely unrelated to the stated PR's purpose.

Comment (abuts) [reply horace_core/sqw/@sqw/head.m:1]: well, I may reword the ticket as code quality changes as the bug here only with old code (which is not running anyway -- just bug a bit werid)
